Kconfig

```diff
@@ -0,0 +1 @@
+rsource "drivers/Kconfig"
```

Comment:
Similarly a comment here would be good.

Also please add copyright and license throughout the PR.